css/index.css

```diff
@@ -1,3 +1,185 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+  padding-left: 100px;
+  padding-top: 30px;
+}
+
+header {
+  border: 1px solid blue;
+  display: flex;
+}
+
+nav {
+  display: flex;
+  justify-content: space-evenly;
+  border: 1px solid red;
+  padding-left: 10px;
+  width: 80%;
+}
+
+
+.top-sec {
+  border: 1px solid black;
+  display: flex;
+  align-items: center;
+  font-size: 60px;
+  justify-content: space-around;
+}
+
+.IOD {
+  align-content: center;
+  border: 1px solid pink;
+  text-align: center; 
+} 
+
+  button {
+    width: 60%;
+    height: 2rem
+  }
+
+.mid-sec-top {
+  display: flex;
+}
+
+.middle-img {
+  width: 100%;
+  max-height: 70%;
+}
+
+.bot-sec {
+  display: flex;
+}
+
+footer {
+  text-align: center
+}
+
+@media(max-width: 800px) {
+    body {
+        padding: 3%;
+        width: 800px;
+        background: blueviolet;
+    }
+
+    header {
+        flex-direction: column-reverse;
+        align-items: center;
+    }
+
+    .logo {
+        width: 80%;
```

Comment:
The logo won't scale well in its current format; will get distorted at a larger size. Not necessary to match the design file.